.goreleaser.yml

```diff
@@ -2,7 +2,7 @@ project_name: lakeFS
 before:
   hooks:
     - go mod download
-    - make gen
+    - make gen PACKAGE_VERSION={{ .Version }}
```

Comment:
You use `${{ steps.version.outputs.tag }}` in `pypi.yaml`

Reply:
Right, this was part of a test to use gorelease to publish the package - will remove it